operators/go/exapmle_subm_ns.yaml

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This should be `example_subm_ns.yaml` (flipped ‘p’ and ‘m’).

Reply:
Done in the latest